include/setup.php

```diff
@@ -496,8 +496,7 @@
 $vars['indexurl'] = $config->getURL('', '', 'index');
 $vars['validationurl'] = getFullURL($_SERVER['SCRIPT_NAME']).'?'.buildQuery($queryParams + array('template' => $template, 'language' => $language), '%26');
 
-// To avoid a possible XSS exploit, need to clean up the passed-in path first
-$path = !empty($_REQUEST['path']) ? escape($_REQUEST['path']) : null;
+$path = !empty($_REQUEST['path']) ? $_REQUEST['path'] : null;
```

Comment:
Removing `escape` without addressing all places where a path is forwarded 1:1 into HTML is problematic, as it will introduce XSS.

Reply:
True, identifying them is hard. Its use is spread all over.